apps/settings/elements/root.html

```diff
@@ -24,37 +24,37 @@ <h2 data-l10n-id="networkAndConnectivity">Network &amp; Connectivity</h2>
           </label>
         </li>
         <li data-category="connectivity">
-          <a id="menuItem-wifi" class="menu-item" data-icon="wifi-4" href="#wifi">
+          <a id="menuItem-wifi" class="menu-item" data-icon="wifi-4" href="#wifi" data-l10n-id="wifiSection" aria-describedby="wifi-desc">
```

Comment:
Unfortunately we can't use aria-hidden on the menu item with an icon because it will make the whole sub-tree invisible to the screen reader. One thing to notice is that if aria-label (which is what data-l10n-id is for) is present for elements with link or role semantics, it automatically overrides what screen reader otherwise would read from the content of the menu item. Which means we would be loosing the description items and thus we need to use aria-describedby explicitly to make sure the screen reader user gets them.